timeseries-streaming/timeseries-python-applications/MLPipelineExamples/sin_wave_example/training/timeseries_local_simple_data.py

```diff
@@ -43,8 +43,9 @@
     trainer_custom_config={'epochs': 30,
```

Comment:
why epochs is needed? above train steps is for iteration count

Reply:
That would set the train steps per epoch. Also is there a derived way to set this with the data from StatsGen stage? So given batch size of x derive number of training steps given epchs of y... Would be nice to have that more automated.

Reply:
I see the pattern used here, does the comp based on the data size: https://github.com/tensorflow/tfx/blob/master/tfx/examples/iris/iris_utils_native_keras.py
I can change the custom values to be one of data size and batch size, and then as per the iris example have the value of epoch become s=int(fn_args.train_steps / steps_per_epoch) where steps_per_epoch is  steps_per_epoch = _TRAIN_DATA_SIZE / _TRAIN_BATCH_SIZE , but pass in the _TRAIN values as custom values.

Reply:
Update in latest commit , please check. 1025KB@

Reply:
epochs has some [issue](https://stackoverflow.com/questions/45989971/distributed-training-with-tf-estimator-resulting-in-more-training-steps) in distributed training, that's why we all use steps in our examples

Reply:
Right now I still populate epoch but its derived from batch / total_sample_size. Should I not set epch at all then?